react/__tests__/LocationSearch.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { render, screen, fireEvent } from '@vtex/test-tools/react'
+
+import LocationSearch from '../LocationSearch'
+import { getFewAddresses } from '../__mocks__/LocationSearchMock'
+
+const UNTABABBLE = -1
+
+describe('Location Search', () => {
+  it('should be able to select a suggested address when a search term is found', async () => {
+    render(<LocationSearch getAddresses={getFewAddresses} />)
+
+    const input = screen.getByLabelText(
+      'Address and house number'
```

Comment:
nitpick comment: you can use regex with the `i` flag so the message can be case-insensitive

Reply:
I prefer keeping it that way because

- it's simpler
- it would fail the test if someone changed the case to something like "Address and House Number"

unless this is a common/best practice and I'm not aware of?

Reply:
but _should_ the test fail only because the casing changed? the end user isn't impacted at all if the casing is "Address and House Number" or "Address and house number", that's why I said about using the case insensitive regex